snap/snapcraft.yaml

```diff
@@ -47,6 +49,3 @@ parts:
       - libgstreamer-plugins-base1.0-0
```

Comment:
Could you also remove libgstreamer-plugins-base1.0-0 and libgudev-1.0-0 since they are in the gnome sdk?